GUI/src/main/kotlin/interface/AbstractFrameNavigationInterface.kt

```diff
@@ -0,0 +1,30 @@
+
+/**
+ * Provides navigation methods for frames in a video.
+ *
+ * This interface defines methods to navigate frames in various ways, including jumping a specified number
+ * of frames in a given direction, jumping to a specific percentage of the video, jumping to a specific frame,
+ * and jumping to the next difference frame. Implementing classes should provide implementation for these methods.
+ */
+interface AbstractFrameNavigationInterface {
```

Comment:
I think, you can leave out the Abstract in the name, possibly also the Interface, but we dont really have consistent conventions atm.

Reply:
I left Interface in the title. I like to see the purpose/type of the class also in the filename and I think per convention the class has to have the same name as its file.